ci/dev.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:12
+
+RUN apt-get update && \
+    apt-get install -y curl iproute2 vim libgpgme-dev libdevmapper-dev iptables net-tools libsecret-1-dev libx11-dev libxkbfile-dev libxkbfile-dev
```

Comment:
Put each of these on their own line.

Reply:
Why do we need libdevmapper-dev?

Reply:
Also libxkbfile-dev is duplicated

Reply:
Why not reduce the layering?

Reply:
I don't mean apt-get install for each, just the package names multiline. See the other docker files in the ci directory.